.github/workflows/ci.yml

```
@@ -94,6 +96,46 @@ jobs:
           ./assert-non-empty-json "http://localhost:8004/search.php?q=avenue%20pasteur"
           docker stop nominatim
 
+      - name: Check UPDATE_MODE=once with volume
+        if: ! matrix.nominatim.version == "4.0"
```

```suggestion
        if: matrix.nominatim.version != "4.0"
```
GH complains about an invalid syntax. I think the correct syntax is something like this. Or we need the curly braces.